video_xblock/backends/brightcove.py

```diff
@@ -406,6 +399,26 @@ def get_frag(self, **context):
         )
         return frag
 
+    def get_player_html(self, **context):
+        """
+        Add VideoJS plugins to the context and render player html using base class logic.
+        """
+        context.update({
+            'videojs_transcript_plugin': self.resource_string(
+                'static/bower_components/videojs-transcript/dist/videojs-transcript.js'
+            ),
+            'videojs_transcript': self.resource_string(
+                'static/js/videojs-transcript.js'
+            ),
+            'videojs_offset': self.resource_string(
+                'static/bower_components/videojs-offset/dist/videojs-offset.js'
+            ),
+            'videojs_speed_handler': self.resource_string(
+                'static/js/videojs-speed-handler.js'
+            )
+        })
+        return super(BrightcovePlayer, self).get_player_html(**context)
```

Comment:
I think it is better to create one context variable which contains list of js files.
And render it inside the loop.
In this case we will avoid adding new js files in both places (brightcove.py and brightcove.html)